File: MAP/Seminar11/Curs12_2019_2019/Curs12_2019_2019/Model/Validator/MessageTaskValidator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Curs12.Model;
namespace Curs12.Model.Validator
{
    class MessageTaskValidator : IValidator<MessageTask>
    {
        public void Validate(MessageTask e)
        {
            bool valid = true;
            if (valid == false)
            {
                throw new ValidationException("Obiectul nu e valid");
            }
        }
    }
}

File: MPP/Curs1/Curs1_Test/Test/src/main/java/domain/MessageTaskValidator.java
package domain;

import domain.MessageTask;
import domain.ValidationException;
import domain.Validator;

/**
 * Created by camelia on 10/16/2017.
 */
public class MessageTaskValidator  implements Validator<MessageTask> {
    @Override
    public void validate(MessageTask entity) throws ValidationException {
        String errMsg="";

        if (errMsg!="")
            throw new ValidationException(errMsg);
    }
}
